platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -713,25 +715,40 @@ private static void updateLoadedStateAddressBook(final SignedState signedState,
     }
 
     /**
-     * Load the state from the disk if it is present.
+     * Load the signed state from the disk if it is present.
+     *
+     * @param configAddressBook the address book to use for emergency recovery.
+     * @return the signed state loaded from disk.
      */
-    private LoadedState loadSavedStateFromDisk() {
+    private SignedState getUnmodifiedSignedStateFromDisk(final AddressBook configAddressBook) {
```

Comment:
wasnt this moved to the browser?

Reply:
yes, I forgot to delete it.  Thanks for the catch!